### DISCUSSION TITLE
Default spacing

### DISCUSSION THREAD

**Discussion by AnnaYuS** [tailwindlabs/tailwindcss#2614] [Help] [answered]
Default spacing in the default config file https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js
doesn't match the defaults in the docs https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale

Is it possible to use spacing `9: '2.25rem',` and other values that are missing in the docs?

**Comment by simonswiss** [comment]
Hey @AnnaYuS 👋 

[These were literally added 14 hours ago](https://github.com/tailwindlabs/tailwindcss/commit/dc9d510c8ac9d071cc8f6d9987df58868ea976a3#diff-d65cc1efb133c1202d14550da74ce17790b33ca9c27dec2a4ce64603d606dca0), so it's likely that there may be discrepancies between the config file and the docs.

You can use the default config as the source of truth for now, until the documentation catches up 😅

**Reply by simonswiss** [reply]
@AnnaYuS please [take note of Adam's comment](https://github.com/tailwindlabs/tailwindcss/discussions/2614#discussioncomment-105095) - I was a little too quick with the reply, and indeed, while these values are in the master branch already, they haven't been _released_ yet, so they won't work when pulling Tailwind CSS from the npm package registry.

My bad! 😅

At least you have a sneak peek of what's coming in a few weeks 👍

**Reply by AnnaYuS** [reply]
@simonswiss Got it! No worries 😄

**Comment by AnnaYuS** [comment]
Thanks @simonswiss 😄

**Comment by adamwathan** [comment, marked as answer]
Can use those values by default yet because we haven’t released a new version. Next release is 2.0 which will be out in a couple weeks 👍🏻
